src/styles/main.scss

```diff
@@ -10,5 +10,76 @@ body {
   box-sizing: border-box;
 }
 
-@import "_header.scss";
-@import "_catalog.scss";
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+  justify-content: center;
+  align-items: center;
+
+  @media (min-width: 768px) and (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
+  }
```

Comment:
Use one condition at a time by putting media queries in the correct order.
Example:
default -> media (max-width: 1023px) -> media (max-width: 767px) -> media (max-width: 477px)